clin/run.py

```diff
@@ -99,24 +139,42 @@ def apply(
     default=False,
     help="Show Nakadi payload (default - false)",
 )
+@click.option(
+    "-i",
+    "--id",
+    required=False,
+    type=str,
+    multiple=True,
+    help="Select one or multiple steps to process by their id",
+)
+@click.option(
+    "-e",
+    "--env",
+    required=False,
+    type=str,
+    multiple=True,
+    help="Select one or multiple steps to process by mathing the target environment",
+)
 @click.argument("file", type=click.Path(exists=True, dir_okay=False, readable=True))
 def process(
-    token: Optional[str],
-    execute: bool,
+    token: str,
     verbose: bool,
+    execute: bool,
     show_diff: bool,
     show_payload: bool,
+    id: Tuple[str],
```

Comment:
```suggestion
    ids: Tuple[str],
```

Reply:
Not sure about this one. The command syntax is like this:

```
clin apply --id xxx event.yaml
clin apply --id xxx --id yyy --id zzz event.yaml
```

I'm not sure if there is a commonly accepted pattern for this, but to me if it was `--ids` I would probably expect to be able to pass a csv or something like that. Maybe like `--ids xxx,yyy,zzz`. For what it's worth, the [click docs](https://click.palletsprojects.com/en/7.x/options/#multiple-options) also uses singular.

Reply:
I'm just saying, that variable that refers to the collection should be in plural form.
It is not about cli API, it is about function params naming only.

Reply:
Oh ok, I see, that would make sense. Unfortunately that not possible though; the arg names must match the click API

Reply:
Ah! Indeed. Sorry for the confusion.